test/e2e/validators.go

```diff
@@ -240,30 +245,37 @@ func validateBuildRunToFail(
 		err = clientGet(buildRunNsName, testBuildRun)
 		Expect(err).ToNot(HaveOccurred(), "Error retrieving build run")
 
-		Expect(testBuildRun.Status.Succeeded).ToNot(Equal(trueCondition))
+		if testBuildRun.Status.GetCondition(operator.Succeeded) == nil {
+			return corev1.ConditionUnknown
+		}
+
+		Expect(testBuildRun.Status.GetCondition(operator.Succeeded).Status).ToNot(Equal(trueCondition))
 
 		now := time.Now()
 		if now.After(nextStatusLog) {
 			Logf("Still waiting for build run '%s' to fail.", testBuildRun.Name)
 			nextStatusLog = time.Now().Add(60 * time.Second)
 		}
 
-		return testBuildRun.Status.Succeeded
+		return testBuildRun.Status.GetCondition(operator.Succeeded).Status
 	}, time.Duration(550*getTimeoutMultiplier())*time.Second, 5*time.Second).Should(Equal(falseCondition), "BuildRun did not fail")
 
 	// Verify that the BuildSpec is available in the status
 	Expect(testBuildRun.Status.BuildSpec).ToNot(BeNil())
 
 	// Verify the build run failure
-	Expect(testBuildRun.Status.Reason).To(MatchRegexp(expectedReasonRegexp))
+	Expect(testBuildRun.Status.GetCondition(operator.Succeeded).Message).To(MatchRegexp(expectedReasonRegexp))
```

Comment:
Makes probably sense to rename the variable to `expectedMessageRegexp`.

Reply:
+1, I will change this.